src/app/myProfile/EmptyUserProfile.ts

```diff
@@ -0,0 +1,8 @@
+export const emptyUserProfile = {
+  firstName: '-',
+  lastName: '-',
+  username: '-',
+  location: '-',
```

Comment:
IMO it would be better just to leave these fields empty, because then if you want to update it, first you need to remove the dashes

Reply:
I added them intentionaly, since the empty profile with just "" looks bad in my opinion:
![image](https://github.com/user-attachments/assets/330fa7c7-ead3-4a09-b362-f3d7b01813e1)
![image](https://github.com/user-attachments/assets/4ff2d454-c967-4510-829c-e32e2649ec31)

Reply:
Yea but with them it also does not look good IMO, anyway I will approve and you can merge if you don't want to change that ;)